src/software/thunderscope/replay/replay_controls.py

```diff
@@ -152,14 +152,14 @@ def refresh(self):
             else "Start\nClip"
         )
 
-    def keyPressEvent(self, event):
+    def keyPressEvent(self: "ReplayControls", event) -> None:
```

Comment:
The `event` param should have a type annotation. You'll need to import `from pyqtgraph.Qt import QtGui` and then
```suggestion
    def keyPressEvent(self, event: QtGui.QKeyEvent) -> None:
```